src/test/kotlin/javabot/operations/ChatGPTOperationTest.kt

```diff
@@ -0,0 +1,39 @@
+package javabot.operations
+
+import com.google.inject.Inject
+import javabot.BaseTest
+import javabot.Javabot.Companion.LOG
+import javabot.JavabotConfig
+import org.testng.Assert.assertTrue
+import org.testng.annotations.Test
+
+class ChatGPTOperationTest : BaseTest() {
+    @Inject
+    protected lateinit var operation: ChatGPTOperation
+
+    @Inject
+    protected lateinit var config: JavabotConfig
+
+    @Test
+    fun testSwallow() {
```

Comment:
i had to read the body of this message to reframe this method name in my head.  :)